### PR TITLE
Update My Initiatives

### PR DESCRIPTION
**Pull request by kevinyu05**

### CODE REVIEW

File: spec/features/initiative_spec.rb [hunk 1]
```diff
@@ -20,4 +20,22 @@
...
+    visit edit_initiative_path(@initiative)
+    expect(page).to have_content('Edit Initiative')
+  end
+  scenario 'change initiative name and target date' do
+    visit edit_initiative_path(@initiative)
+    fill_in 'Initiative Name', with: 'TestB'
+    fill_in 'Target Date', with: '01/01/2020'
+    fill_in 'Description', with: 'TestJ'
+    click_button 'Submit'
+    expect(page).to have_content('TestB')
```
Comment (jasononaquest) [inline spec/features/initiative_spec.rb:40]: Does it seem like you might want to check for 'TestA' to make sure it's not there?

File: app/views/initiatives/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<div class='wrapper'>
+  <h2>Edit Initiative </h2>
+  <div class='card full-width'>
+  <div class='w-form'>
+    <%= form_for @initiative do |f| %>
```
Comment (jasononaquest) [inline app/views/initiatives/edit.html.erb]: I think there's an opportunity here to extract the form to a partial since it's used in both #new and #edit.

**Review by jasononaquest — COMMENTED** [review]
This looks great! Just a couple suggestions for your consideration.

File: spec/features/initiative_spec.rb (hunk 1 above)
Comment (kevinyu05) [reply spec/features/initiative_spec.rb:40]: We checked for 'TestA' to make sure it's not there

File: app/views/initiatives/edit.html.erb (hunk 2 above)
Comment (kevinyu05) [reply app/views/initiatives/edit.html.erb]: We've extracted the form to be a partial

**Pull request merged**